data_steward/utils/pipeline_logging.py

```diff
@@ -16,27 +16,46 @@
 
 # Project imports
 import resources
+from app_identity import get_application_id
 
 DEFAULT_LOG_DIR = os.path.join(resources.base_path, 'logs')
 """Default location for log files"""
 DEFAULT_LOG_LEVEL = logging.INFO
 
 LOG_FORMAT = '%(asctime)s - %(levelname)s - %(name)s - %(message)s'
 LOG_DATEFMT = '%Y-%m-%d %H:%M:%S'
-FILENAME_FMT = '%Y%m%d'
+LOG_FILENAME_DATEFMT = '%Y%m%d'
 
 _FILE_HANDLER = 'curation_file_handler'
 """Identifies the file log handler"""
 _CONSOLE_HANDLER = 'curation_console_handler'
 """Identifies the console handler"""
 
 
-def _get_date_str():
+def _get_log_date_str():
     """
-    Get current date formatted using FILENAME_FMT
+    Get current date formatted using LOG_FILENAME_DATEFMT
     :return: 
     """
-    return datetime.today().strftime(FILENAME_FMT)
+    return datetime.today().strftime(LOG_FILENAME_DATEFMT)
+
+
+def _get_log_filename() -> str:
+    """
+    Construct runtime-specific log filename
+    """
+    try:
+        # attempt to add application id suffix (google project name)
+        app_id = get_application_id()
+        filename_suffix = f'-{app_id}'
+    except Exception:
```

Comment:
I'd suggest explicitly catching `RuntimeError`, since that's what `get_application_id()` might raise.